WaterLog_Backend/Controllers/SensorHistoryController.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using WaterLog_Backend.Models;
+
+namespace WaterLog_Backend.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SensorHistoryController : ControllerBase
+    {
+        private readonly DatabaseContext _db;
+        readonly IConfiguration _config;
+        private IControllerService _service;
+
+        public SensorHistoryController(DatabaseContext context, IConfiguration config, IControllerService service)
+        {
+            _db = context;
+            _config = config;
+            _service = service;
+        }
+
+        // GET api/sensor history
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<SensorHistoryEntry>>> Get()
```

Comment:
More descriptive end point function names please

Reply:
Applies to all endpoints

Reply:
These are auto-generated but yes I agree with Fred. So lets say this one gets sensor history you can rename it to `GetSensorHistory` OR `GetHistory` and the one below this one getting by ID can be renamed to `GetSensoryHistoryById` or `GetHistoryById`

Reply:
Changed function names.